CHANGES.rst

```diff
@@ -1,6 +1,8 @@
 0.3.0 (unreleased)
 ==================
 
+- Honor the ``collect_ignore`` option used in ``conftest.py``. [#34]
```

Comment:
Just a nit, but we need to change the PR number here now.

Reply:
Okay, I've just fixed that and force-pushed.